task/buildpacks-phases/0.1/buildpacks-phases.yaml

```diff
@@ -76,6 +79,18 @@ spec:
       securityContext:
         privileged: true
 
+    - name: copy-stack-toml
```

Comment:
Can we add a comment explaining why this is necessary?

Reply:
I feel a little mixed on this. We don't have any comments around the `prepare` step either (though perhaps, that could also be added). I can add the comment, but it appears a little odd. Maybe it's okay though - what do you think?

Reply:
Hmm I hear that. I vote for more comments, though 🤓  As a buildpacks consumer, I may be able to understand that there are these phases doing 'app' things, but I'd have no idea why they are doing these other steps, and whether they are part of the normal procedure. Especially if this is meant to serve as an example to people to show what the lifecycle does, it would be nice to differentiate essential lifecycle functionality from steps we take to prepare, and to be explicit about it.

Reply:
I second the vote (although not as a blocker) for more comments given that we typically use the tekton tasks as a good reference when asking other platforms to support CNB.